### ISSUE TITLE
Logo Options

### ISSUE THREAD

**Issue by wpalchemist** [wpalchemist/pressable-trial#1]
I have put together some rough sketches of three possible logos.  Does any one of these stand out?  Would you like to use one on the site?  I can swap colors if you like the colors of one but the birds of another.  If none of these appeal to you, let me know and I can go back to the drawing board.  
![logo_draft_3](https://user-images.githubusercontent.com/16658165/38582211-e1b0138e-3cc3-11e8-865b-77ecdd8365df.png)
![logo_draft_1](https://user-images.githubusercontent.com/16658165/38582214-e1c6eb36-3cc3-11e8-83bc-612009cf548c.png)
![logo_draft_2](https://user-images.githubusercontent.com/16658165/38582216-e1dced50-3cc3-11e8-86b5-9c34cdaeb1dd.png)

**Comment by threegreenbirds** [comment]
These are amazing! Thank you. Our CEO likes the middle logo where all of the birds are sitting on the same level but the rest of us like the third one, the colors are really great. Maybe you could combine them somehow?

What do you think about animating the tails and beaks?

**Comment by wpalchemist** [comment]
Since you liked the second of the two color schemes I recommended, I can do the birds in a row but with the other color scheme.  

I can animate the birds, but before we go that route, we should think about why we want animation here.  Animation can be really nifty and fun, but we want to make sure that it serves a purpose.  For instance, animation can be used to illustrate a point, tell a story, or provide feedback to a user about interaction (such as a loading animation).  

Animating the birds would be neat, but I'm afraid it would be a distraction.  The logo is going to be in the header, at the very top of the site.  Honestly, that's not a place where we want users to spend a lot of time.  We want them to scroll down and start reading the page content and make donations, so encouraging them to watch an animated logo will only distract them from more important business. If we did do an animation there, I would want it to be short and subtle, and only happen under certain circumstances (like when you first load the page, or when you hover over the logo).  But even then, the animation doesn't really serve much purpose other than looking nifty.  

Of course, there's nothing wrong with looking nifty, so if animation is something you really want to do, I'm happy to do it.  I just want to make sure that you are aware that it doesn't serve your site goals of sharing information and collecting donations, and might be distracting to your users.
